docs/wsrep-system-index.md

```diff
@@ -781,6 +781,26 @@ You can set it to any value between `1024` and the default `2147483647`.
 
     [MySQL wsrep option: wsrep_max_ws_size](https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-max-ws-size)
 
+### `wsrep_mode`
+
+| Option         | Description        |
+| -------------- | ------------------ |
+| Command Line:  | ``--wsrep-mode``   |
+| Config File:   | Yes                |
+| Scope:         | Global             |
+| Dynamic:       | Yes                |
+| Default Value: |                    |
+
+This variable has been implemented in [Percona XtraDB Cluster 8.0.31](./release-notes/8.0.31-23.md).
+
+Defines the node behavior according to a specified value. The variable is disabled by default.
```

Comment:
It has two possible values: empty and IGNORE_NATIVE_REPLICATION_FILTER_RULES
So - does empty = disabled?

Reply:
updated